### PR TITLE
Fix webadmin_create_group_and_user tests

### PR DESCRIPTION
**Pull request by dominikl**
# What this PR does

With the 'roles' branch merged in, the UI of the webadmin's group edit dialog changed a bit.
This PR updates the robot tests accordingly.

# Testing this PR

Check robotframework tests: https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-robotframework/

Note: Ignore flaky tests and 'Drag and Drop Image/Plate' tests, the tests to look out for regarding this PR are: 'Check Admin Edit Self' and 'Check User Form'

### CODE REVIEW

File: components/tests/ui/testcases/web/webadmin_create_group_and_user.txt [hunk 1]
```diff
@@ -142,9 +145,8 @@ Check Admin Edit Self
     Go To                             ${USERS URL}
     Click Element                     xpath=//table[@id="experimenterTable"]/tbody/tr[descendant::td[contains(text(), '${ROOT FULL NAME}')]]//a[contains(@class, "btn_edit")] 
 
-    Wait Until Page Contains Element  id=id_administrator       ${WAIT}
-    Checkbox Should Be selected       id=id_administrator
-    Element Should Be Disabled        id=id_administrator
+    Wait Until Page Contains Element  xpath=//input[@type="radio" and @value="administrator"]       ${WAIT}
+    Radio Button Should Be Set To     role  administrator
```
Comment (mtbc) [inline components/tests/ui/testcases/web/webadmin_create_group_and_user.txt:149]: odd double space, is it usual for these? I figured you were vertically lining up with something but I can't see what

**Comment by mtbc** [comment]
Will relist for tomorrow when we might have a happier Robot Framework job.

File: components/tests/ui/testcases/web/webadmin_create_group_and_user.txt (hunk 1 above)
Comment (dominikl) [reply components/tests/ui/testcases/web/webadmin_create_group_and_user.txt:149]: Thanks Mark. I'll check the formatting of the other robot tests again, how it's done there.

**Comment by mtbc** [comment]
Passes at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-robotframework/lastCompletedBuild/robot/Web%20&%20Web/Web_1/Webadmin%20Create%20Group%20And%20User/.

**Review by mtbc — APPROVED** [review]

File: components/tests/ui/testcases/web/webadmin_create_group_and_user.txt (hunk 1 above)
Comment (dominikl) [reply components/tests/ui/testcases/web/webadmin_create_group_and_user.txt:149]: I just leave it for now. The indention in the robot tests is not really consistent anyway.

**Pull request merged**